grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -713,9 +748,15 @@ public GrpcService build() {
                     "'httpJsonTranscodingErrorHandler' can only be set if HTTP/JSON transcoding feature " +
                     "is enabled");
         }
+        if (enableHealthCheckService) {
+            grpcHealthCheckService = GrpcHealthCheckService.builder()
+                                                           .build();
+        }
+        if (grpcHealthCheckService != null) {
+            registryBuilder.addService(grpcHealthCheckService.bindService());
```

Comment:
noted that `GrpcHealthCheckService` is also decorated by grpc interceptors ✅

Reply:
@jrhee17 
Should I write it in the Javadoc?